README.md

```diff
@@ -215,8 +215,8 @@ The `liberty-plugin-archetype` is used to generate a basic multi-module project
     mvn archetype:generate \
         -DarchetypeGroupId=io.openliberty.tools \
         -DarchetypeArtifactId=liberty-plugin-archetype \
-        -DarchetypeVersion=2.2 \
-        -DlibertyPluginVersion=3.2 \
+        -DarchetypeVersion=3.2.2  \
```

Comment:
Let's go ahead and change all references in doc and code to 3.2.3 since we know we fixed that NPE last week.